samples/java/src/main/java/com/contoso/kafka/TriggerFunction.java

```diff
@@ -32,23 +32,23 @@ public void runOne(
      * @param kafkaEventData
      * @param context
      */
-    @FunctionName("KafkaTrigger-Java-Many")
-    public void runMany(
-            @KafkaTrigger(
-                topic = "message", 
-                brokerList="%BrokerList%",
-                consumerGroup="$Default", 
-                username = "%ConfluentCloudUsername%", 
-                password = "%ConfluentCloudPassword%",
-                authenticationMode = BrokerAuthenticationMode.PLAIN,
-                protocol = BrokerProtocol.SASLSSL,
-                sslCaLocation = "confluent_cloud_cacert.pem", // Remove this attribute for Linux/Mac users.
-                cardinality = Cardinality.MANY,
-                dataType = "string"
-             ) String[] kafkaEventData,
-            final ExecutionContext context) {
-            for (String message: kafkaEventData) {
-                context.getLogger().info(message);
-            }    
-    }
+    // @FunctionName("KafkaTrigger-Java-Many")
```

Comment:
Why did we comment it?

Reply:
We have three sample functions. `KafkaOutput`, `Kafka-Trigger-Java`, and `Kafka-Trigger-Java-Many`. The `Kafka-Trigger-Java-Many` is the sample for Cardinality.MANY and Confluent Cloud. So not many client don't have the Confluent Cloud account. Instead, for the others, it accesses to the local cluster that the DevContainer automatically starts. 
That is why I comment on the `Kafka-Trigger-Java-Many` not to throw an error if the customer doesn't configure Confluent Cloud settings, but I wanted to provide some complex use-case samples.